lib/fs.js

```diff
@@ -376,6 +376,16 @@ fs.exists = function(path, callback) {
   }
 };
 
+Object.defineProperty(fs.exists, internalUtil.promisify.custom, {
+  enumerable: false,
```

Comment:
AFAIK it's supposed to be "hidden" from userland.
See [lib/fs.js#L780](https://github.com/nodejs/node/blob/master/lib/fs.js#L780)